src/components/UserProfile/Bio.jsx

```diff
@@ -54,7 +55,13 @@ function Bio({
           </React.Fragment>
         ),
         empty: (
-          <EmptyContent onClick={() => onEdit('bio')}>Add a short bio</EmptyContent>
+          <EmptyContent onClick={() => onEdit('bio')}>
+            <FormattedMessage
+              id="profile.bio.empty"
+              defaultMessage="Add a short bio"
+              description="instructions when the user hasn't written an About Me"
+            />
+          </EmptyContent>
```

Comment:
Is FormattedMessage the way we intend to add any localized string? (Even for unformatted messages?)